runtime/queries/scala/injections.scm

```diff
@@ -1,2 +1,16 @@
-((comment) @injection.content
+([(comment) (block_comment)] @injection.content
  (#set! injection.language "comment"))
+
+
+; TODO for some reason multiline string (triple quotes) interpolation works only if it contains interpolated value
+; Matches these SQL interpolators:
+;  - Doobie: 'sql', 'fr'
+;  - Quill: 'sql', 'infix'
+;  - Slick: 'sql', 'sqlu'
+(interpolated_string_expression 
+  interpolator: 
+    ((identifier) @interpolator 
+     (#match? @interpolator "^(fr|infix|sql|sqlu)$"))
```

Comment:
After https://github.com/helix-editor/helix/pull/9317 the `#any-of?` predicate is supported.

```suggestion
     (#any-of? @interpolator "fr" "infix" "sql" "sqlu"))
```

Reply:
True, now changed!